code/modules/projectiles/projectile/energy/stun.dm

```diff
@@ -28,3 +31,9 @@
 /obj/projectile/energy/electrode/on_range() //to ensure the bolt sparks when it reaches the end of its range if it didn't hit a target yet
 	do_sparks(1, TRUE, src)
 	..()
+
+/obj/projectile/energy/electrode/turret
+	name = "low-wavelength electrode" //the low wavelengths make it not deal any amount of stamina damage, but instead paralyze the victim. this is how it works in real life

```

Comment:
high power electrode?

Reply:
it's for turrets only, works just like the old electrode. long stun, no convulsions, no stamina damage

Reply:
I meant the name, wavelength doesn't quite fit the bill?